processor/spancountprocessor/README.md

```diff
@@ -0,0 +1,83 @@
+# Span Count Processor
+This processor is used to convert the number of spans received during an interval into a metric.
+
+## Supported pipelines
+- Traces
+
+## How It Works
+1. The user configures the span count processor in their traces pipeline and a route receiver in their target metrics pipeline.
+2. If any incoming metrics match the `match` expression, they are counted and dimensioned by their `attributes`. Regardless of match, all metrics are sent to the next component in the metrics pipeline.
```

Comment:
Should this say `spans` instead of `metrics`?

Reply:
Yep, it sure should!